docs/TVM_EP.md

```diff
@@ -1,33 +1,34 @@
-# Standalone TVM (STVM) Execution Provider
+# TVM Execution Provider
 
 ## Contents
 
 - [Introduction](#introduction)
-- [Build](#build)
-- [Configuration options](#configuration-option)
+- [Build](#build-onnx-runtime-with-the-tvm-execution-provider)
+- [Configuration options](#configuration-options)
 - [Performance Tuning](#performance-tuning)
 - [Samples](#samples)
 - [Known issues](#known-issues)
 
 
 ## Introduction
 
-STVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
-STVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
+TVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
+TVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
 
-### Build ONNX Runtime with the STVM Execution Provider
+## Build ONNX Runtime with the TVM Execution Provider
 
 Install the minimal pre-requisites on Ubuntu/Debian like linux operating systems:
-```
+```bash
 apt-get install -y python3 python3-dev python3-pip python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev llvm-12
 pip3 install numpy decorator attrs
 ```
 
-Clone this repo.
+If you want to use the `GPU` then make sure you have installed NVidia driver and CUDA Toolkit.
```

Comment:
If TVM has specific requirement on the version, is there a page the documentation can link to?

Reply:
Hello @xadupre!
There are no special version requirements. 
There are also no additional links to documentation. CUDA is an external dependency in this context, so it is up to the user to decide how they will deliver this dependency.

Reply:
Internally we're using CUDA 10, but moving over to CUDA 11.1 and ensuring this leads to no performance regressions.